pkg/server/sandbox_run_windows.go

```diff
@@ -362,6 +364,14 @@ func (c *criService) generateSandboxContainerSpec(id string, config *runtime.Pod
 		g.AddAnnotation(k, v)
 	}
 
+	if _, ok := config.Annotations[annotations.VMMemorySizeInMB]; !ok {
```

Comment:
This is the opposite of putting them in the toml. We dont need this because the entire opts struct is forwarded at create. and you handle an annotation passed at that time as an override so this PR doesnt actually do anything.

Reply:
Hmm. I see. I will send out a PR in containerplatform repo with default values for these in toml file then

Reply:
Thanks!